.circleci/config.yml

```diff
@@ -66,6 +66,10 @@ jobs:
           command: yarn run test:unit
           no_output_timeout: 120
 
+      - run:
```

Comment:
move it to separate check, so on github we'll see this as a separate failure/succeding test